src/BenchmarkDotNet/Running/BenchmarkRunnerClean.cs

```diff
@@ -614,6 +620,15 @@ private static void Cleanup(HashSet<string> artifactsToCleanup)
             }
         }
 
+        private static void UpdateTitle(in StartedClock runsChronometer, int totalBenchmarkCount, int benchmarksToRunCount)
+        {
+            int executedBenchmarkCount = totalBenchmarkCount - benchmarksToRunCount;
+            double avgSecondsPerBenchmark = runsChronometer.GetElapsed().GetTimeSpan().TotalSeconds / executedBenchmarkCount;
+            TimeSpan fromNow = TimeSpan.FromSeconds(avgSecondsPerBenchmark * benchmarksToRunCount);
+
+            Console.Title = $"{executedBenchmarkCount}/{totalBenchmarkCount} Benchmarks runned - Estimated {(int)fromNow.TotalHours}h {fromNow.Minutes}m to finish";
```

Comment:
Seems kind of long (and "runned" is not a word).

How about `1/16 - 0h 32m - Benchmarks`?

I think it should be # currently executing instead of # complete, but that's just my opinion.

Also, the console title should be reverted after the benchmarks are complete.

Reply:
Sorry for bad english.

I have changed the title a bit, number of remaining benchmarks + time to finish

Also refactored small code duplication